### PR TITLE
Acebook wall

### PR DESCRIPTION
**Pull request by rachjgriff**
3 new features tests to test Acebook Wall
Setup new route in config/routes.rb
Setup new view in posts/display_all.html.erb
Setup new method in PostsController
Added some additional links on various pages to access All Posts, Your Posts and Sign Out

### CODE REVIEW

File: app/views/posts/display_all.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<h1> Acebook Wall </h1>
+
+<% @all_posts.each do |post| %>
```
Comment (Danielandro) [inline app/views/posts/display_all.html.erb]: Dope work

File: app/views/root/index.html.erb [hunk 2]
```diff
@@ -3,8 +3,11 @@
 <% if user_signed_in? %>
   <h2> You have successfully logged in user: <%= current_user.email %></h2>
   <h3> Click the link below to see your posts </h3>
+  <br>
   <%= link_to "View Your Posts", user_posts_path(current_user) %>
   <br>
+  <%= link_to "View All Posts", posts_path %>
```
Comment (Danielandro) [inline app/views/root/index.html.erb]: I like this

File: config/routes.rb [hunk 3]
```diff
@@ -6,4 +6,6 @@
   end
   # setup for devise we should update this when we have a user#index
   root "root#index"
+
+  get 'posts', to: 'posts#display_all'
```
Comment (Danielandro) [inline config/routes.rb]: I like this too

File: spec/features/user_can_view_all_posts_spec.rb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    fill_in("post_message", with: "Hello")
+    click_button "Submit"
+    click_link "View All Posts"
+    expect(current_path).to eq("/posts")
+    expect(page).to have_content("Hello")
+  end
+
+  scenario "User can see other user's posts as well as their own" do
+    sign_up("user@gmail.com")
+    click_link "View Your Posts"
```
Comment (Danielandro) [inline spec/features/user_can_view_all_posts_spec.rb]: I think there is a `create_a_post` helper method in the web_helpers file which could make this more concise

Comment (rachjgriff) [reply spec/features/user_can_view_all_posts_spec.rb]: Nice one Danny - we've updated the tests with this

**Pull request merged**